bc_events/utils.py

```diff
@@ -18,3 +23,14 @@ def build_topic_name(category, entity, action):
         Formatted topic name
     """
     return "{category}.{entity}{action}".format(category=category, entity=entity, action=action)
+
+
+def requests_session():
+    """For use on AWS APIs that periodically fail with 500 and suggest retrying"""
+
+    session = requests.Session()
+    retry = Retry(total=3, read=3, connect=3, status_forcelist=[500], backoff_factor=0.3, method_whitelist=False)
+    adapter = HTTPAdapter(max_retries=retry, pool_maxsize=20, pool_connections=20)
```

Comment:
Do we need to specify `pool_maxsize` or `pool_connections` here? We did in BriteEvents' stack tests because we were hammering it with async requests. I think here, we will only be making a few at a time.